sklearn/neighbors/_classification.py

```diff
@@ -172,6 +173,9 @@ class KNeighborsClassifier(KNeighborsMixin, ClassifierMixin, NeighborsBase):
     [0]
     >>> print(neigh.predict_proba([[0.9]]))
     [[0.666... 0.333...]]
+
+    See :ref:`sphx_glr_auto_examples_neighbors_plot_classification.py`
+    for an example on how to use the API.
```

Comment:
I think that we should put this example reference in the `weights` parameter description.

Indeed, it shows two different classification decision boundaries that change due to this parameter. So I would move and rephrase to:

```python
    Refer to the example entitled
    :ref:`sphx_glr_auto_examples_neighbors_plot_classification.py`
    showing the impact of the `weights` parameter on the decision
    boundary.
```

Reply:
Indeed. I've now moved and rephrased the reference link.